util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -215,15 +217,15 @@ def sanitize_method(method: Optional[str]) -> Optional[str]:
         return method
     return "UNKNOWN"
 
-
-def get_custom_headers(env_var: str) -> List[str]:
-    custom_headers = environ.get(env_var, [])
+def get_custom_headers(env_var: str) -> list[str]:
+    custom_headers = environ.get(env_var, None)
     if custom_headers:
```

Comment:
if you keep the change with None as default, it's better to change the condition: 
```suggestion
    if custom_headers is not None:
```

Reply:
Sure but if it’s `””` there’s no point in taking that branch anyway